src/main/java/io/vertx/core/http/impl/HttpServerImpl.java

```diff
@@ -637,6 +637,7 @@ private void handshake(FullHttpRequest request, Channel ch, ChannelHandlerContex
         ServerWebSocketImpl ws = new ServerWebSocketImpl(vertx, theURI.toString(), theURI.getPath(),
           theURI.getQuery(), new HeadersAdaptor(request.headers()), wsConn, shake.version() != WebSocketVersion.V00,
           connectRunnable, options.getMaxWebsocketFrameSize());
+        ws.metric = metrics.connected(wsConn.metric(), ws);
```

Comment:
it represents a server web socket that connected.

there are two SPI, one for client (HttpClientMetrics.connected with WebSocket), one for server (HttpServerMetrics#connected with ServerWebSocket)